docs/source/en/model_doc/markuplm.mdx

```diff
@@ -95,10 +95,10 @@ This is the simplest case, in which the processor will use the feature extractor
 ...  <title>Hello world</title>
 ...  </head>
 ...  <body>
-
+...
```

Comment:
Need to have this `...` when we want to use `"""`. Otherwise, we get error

```bash
SyntaxError: EOF while scanning triple-quoted string literal
```
when running doctest.